### PR TITLE
ACL_RULE can set ACCEPT in packet action but yang model doesn't support that

### PR DESCRIPTION
**Pull request by wen587**
#### Why I did it
ACL_RULE can set ACCEPT in packet action but yang model doesn't support that
#### How I did it
Add ACCEPT to packet_action
#### How to verify it
`sudo config apply-patch acl.json` will not report failure
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

File: src/sonic-yang-models/yang-models/sonic-types.yang [hunk 1]
```diff
@@ -60,6 +60,7 @@ module sonic-types {
 
     typedef packet_action{
         type enumeration {
+            enum ACCEPT;
```
Comment (qiluo-msft) [inline src/sonic-yang-models/yang-models/sonic-types.yang:63]: Is it only used in control plane ACL? If yes, should we limit the enum used by data plane ACL?

Comment (wen587) [reply src/sonic-yang-models/yang-models/sonic-types.yang:63]: Didn't find any doc talking more about PACKET_ACTION.
[ACL Design](https://github.com/Azure/SONiC/blob/master/doc/acl/ACL-High-Level-Design.mdl) mentioned "FORWARD and DROP" for "L3" tables

**Comment by qiluo-msft** [comment]
Will you close this PR and in favor of https://github.com/Azure/sonic-buildimage/pull/9784?

**Comment by wen587** [comment]
tracked here: https://github.com/Azure/sonic-buildimage/pull/9784

**Pull request closed without merging**